libnavigation-ui/src/test/java/com/mapbox/navigation/ui/map/MapFpsDelegateTest.java

```diff
@@ -75,7 +90,7 @@ public void adjustFpsFor_thresholdSetWithCorrectDuration() {
     when(batteryMonitor.isPluggedIn(any(Context.class))).thenReturn(false);
     MapFpsDelegate delegate = new MapFpsDelegate(mapView, batteryMonitor);
     RouteProgress routeProgress = buildRouteProgressWith(100d, 20d);
-    int maxFps = 5;
+    int maxFps = 30;
```

Comment:
Same as above re: Why was this needed `5` > `30`?

Reply:
Made the change locally to test something but forgot to revert it back. Thanks for flagging. Fixed.